awx/ui_next/src/screens/Template/shared/WorkflowJobTemplateForm.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import { t } from '@lingui/macro';
+import { withI18n } from '@lingui/react';
+import { Formik, Field } from 'formik';
+
+import { Form, FormGroup } from '@patternfly/react-core';
+import { required } from '@util/validators';
+import PropTypes from 'prop-types';
+
+import FormRow from '@components/FormRow';
+import FormField, { FieldTooltip } from '@components/FormField';
+import OrganizationLookup from '@components/Lookup/OrganizationLookup';
+import { InventoryLookup } from '@components/Lookup';
+import { VariablesField } from '@components/CodeMirrorInput';
+import FormActionGroup from '@components/FormActionGroup';
+import ContentError from '@components/ContentError';
+import LabelSelect from './LabelSelect';
+
+function WorkflowJobTemplateForm({
+  handleSubmit,
+  handleCancel,
+  i18n,
+  template = {},
+}) {
+  const [contentError, setContentError] = useState(null);
+  const [inventory, setInventory] = useState(
+    template?.summary_fields?.inventory || null
+  );
+  const [organization, setOrganization] = useState(
+    template?.summary_fields?.organization || null
+  );
+
+  if (contentError) {
+    return <ContentError error={contentError} />;
+  }
+  return (
+    <Formik
+      onSubmit={handleSubmit}
+      initialValues={{
+        name: template.name || '',
+        description: template.description || '',
+        inventory: template.summary_fields?.inventory?.id || '',
+        organization: template.summary_fields?.organization?.id || '',
+        labels: template.summary_fields?.labels?.results || [],
+        variables: template.variables || '---',
+        limit: template.limit || '',
+        scmBranch: template.scm_branch || '',
+        ask_limit_on_launch: template.ask_limit_on_launch || false,
```

Comment:
We actually don't need these yet, prompt on launch is not functional yet, so I'll remove them.

Reply:
@AlexSCorey I think these are fine to leave in.  Shouldn't be any negative impact that I can see.